lib/Form/RadioButton/Other.js

```diff
@@ -22,6 +22,29 @@ class Other extends Component {
   render() {
     const { name, value, id, checked, label, onChangeHandler, onTextChangeHandler } = this.props;
 
+    if (checked) {
```

Comment:
I think this might be missing a test...and so is the contents of the conditional.